incubator/elasticsearch/templates/client-deployment.yaml

```diff
@@ -49,6 +49,9 @@ spec:
 {{- if .Values.client.nodeSelector }}
       nodeSelector:
 {{ toYaml .Values.client.nodeSelector | indent 8 }}
+{{- if .Values.client.tolerations }}
+      tolerations:
+{{ toYaml .Values.client.tolerations | indent 8 }}
```

Comment:
This block needs to be below the next line and it needs it's own end statement.
The rest need the same.